### ISSUE TITLE
Avatar Chooser Dialog

### ISSUE THREAD

**Issue by CoderVishalSehgal** [UdacityAndroidDevScholarship/quiz-app#140]
**By 12 May EOD**

Have to make a Dialog Chooser which contains all these avatars available on the below Link, as per the mockup.
And it should open when clicking on the **Change Profile Picture** on the **User Details Screen**. 
* Choose app ColorPrimaryDark as the background of the Dialog.

**Avatars**-
https://drive.google.com/open?id=15G8rogdKAjSiVmwascp6aYvu97F4XDHv

**Mockup**-
https://github.com/UdacityAndroidDevScholarship/quiz-app/wiki/App-Mockups#avatar-chooser-dialog

**Comment by xsahil03x** [comment]
I would like to work 😄

**Comment by CoderVishalSehgal** [comment]
@xsahil03x yes sure, go ahead 👍

**Comment by SVB-knowmywork** [comment]
I am interested in adding this feature. I will  try to meet the deadline.

**Comment by jainakshansh** [comment]
I'd like to work on this feature.

**Comment by sagarsrao** [comment]
Awesome guys please go ahead.

**Comment by CoderVishalSehgal** [comment]
@xsahil03x @krishna7860 @SVB-knowmywork @jainakshansh Guys I'm so sorry for the interruption, but please hold on for some time, I'll have to make big changes in this issue.

**Comment by YuganshT79** [comment]
I would also like to work on this, @CoderVishalSehgal Please do update when we should start working after you have made the changes 👍

**Comment by nishank95** [comment]
Yeah i would also like to work on this issue and please update us when to start working on it

**Comment by animeshroydev** [comment]
@CoderVishalSehgal I would like to work on it...

**Comment by sayaMahi** [comment]
I shall work on this.
